src/Symfony/Component/Mailer/Bridge/Amazon/composer.json

```diff
@@ -20,7 +20,7 @@
         "symfony/mailer": "^4.4|^5.0"
     },
     "require-dev": {
-        "symfony/http-client": "^4.3|^5.0"
+        "symfony/mailer": "^4.4.18|^5.1.10"
```

Comment:
httpclient is not needed here?
note that mailer is already in require - it should not be duplicated inrequire-dev

Reply:
Sorry I wanted to bump mailer, not http-client, I mixed sth up

Is it correct to bump mailer like that now?

Reply:
nope: the line in "require" should be updated instead